bahdan-pantsialeyeu-products-service/src/functions/handlers.ts

```diff
@@ -0,0 +1,34 @@
+import { ProductsService } from './products.service';
+import * as AWS from 'aws-sdk';
+import { custom } from '../../custom';
+import { v4 as uuid } from 'uuid';
+import {
+  initCreateProduct,
+  validationSchema as createProductValidationSchema,
+} from './createProduct/handler';
+import { initGetProductById } from './getProductById/handler';
+import { initGetProducts } from './getProducts/handler';
+import { initDeleteProduct } from './deleteProduct/handler';
+import {
+  initUpdateProduct,
+  validationSchema as updateProductValidationSchema,
+} from './updateProduct/handler';
+import { middyfy } from '../libs/middlewares';
+import validator from '@middy/validator';
+
+const productsService = new ProductsService(
+  new AWS.DynamoDB.DocumentClient(),
+  custom.productsTableName,
+  uuid
+);
+
+export const createProduct = middyfy(initCreateProduct(productsService)).use(
+  validator({ inputSchema: createProductValidationSchema })
```

Comment:
> +1 (All languages) - POST /products lambda functions returns error 400 status code if product data is invalid

could you please advise if correct error is return if product does not follow the schema